### PR TITLE
Release PyMC3 v3.11.3

### PR DESCRIPTION
**Pull request by canyon289**
Seeing what people think first before proceeding

### CODE REVIEW

**Comment by ricardoV94** [comment]
Do the release notes need to be updated?

**Comment by canyon289** [comment]
> Do the release notes need to be updated?

Definitely. Updated.

Didnt add a v3next section because we dont have one.

Ready for review

**Comment by ricardoV94** [comment]
~~Last minute fix for https://github.com/pymc-devs/pymc3/issues/4950 ?~~ Nevermind, it was not a bug

**Review by Sayam753 — APPROVED** [review]
LGTM

**Comment by Sayam753** [comment]
minions-yay

Thanks @canyon289 
- Can the branches `v3_release` and `v3.11.3` be deleted? to avoid confusion
- And have you pushed `v3.11.3` tag as per the last step on [PyMC3 release checklist](https://github.com/pymc-devs/pymc3/wiki/Release-Checklist)?

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -15,6 +15,11 @@
 [#4720](https://github.com/pymc-devs/pymc3/pull/4720)).
 + `plot_gp_dist` is now available at the package level: `pm.plot_gp_dist` ([see #4835](https://github.com/pymc-devs/pymc3/pull/4835)).
 + The experimental JAX sampling was removed. It will be included in the upcoming `v4` release ([see #4866](https://github.com/pymc-devs/pymc3/pull/4866)).
++ `pm.intX` no longer downcasts integers unnecessarily (see [#4569](https://github.com/pymc-devs/pymc3/pull/4569))
```
Comment (twiecki) [inline RELEASE-NOTES.md:18]: We should always put `New Features` at the top.

Comment (Sayam753) [reply RELEASE-NOTES.md:18]: Good idea. Can you add a line saying the same in Release Checklist - https://github.com/pymc-devs/pymc3/wiki/Release-Checklist?

**Pull request merged**